### DISCUSSION TITLE
(Fedex) Is there a way to get the rates from the PAYOR_ACCOUNT_PACKAGE RateType?

### DISCUSSION THREAD

**Discussion by ocmyhome** [alexeybusygin/ShippingRates#28] [General]
Is there a way to get the rates from the PAYOR_ACCOUNT_PACKAGE RateType (for Fedex)?  From the XML response, there are two RatedShipmentDetails nodes and the library is only returning the rate from the "PAYOR_LIST_PACKAGE" node.  It would be great if all TotalNetCharge > Amount, be returned into the "Rate" object.

![000030](https://user-images.githubusercontent.com/24324485/139733568-6b90421e-85a7-4673-b672-d149cdc29e5a.png)

**Comment by alexeybusygin** [comment]
To get the rates from the PAYOR_ACCOUNT_PACKAGE RateType for FedEx you should set UseNegotiatedRates to true like this:

```
rateManager.AddProvider(new FedExProvider(fedexKey, fedexPassword, fedexAccountNumber, fedexMeterNumber, fedexUseProduction)
{
    UseNegotiatedRates = true
});
```

That would also include results with PAYOR_ACCOUNT_SHIPMENT and NEGOTIATED RateTypes.

**Reply by ocmyhome** [reply]
Awesome, thanks for getting back!  This worked!
